libgraphics/src/context.cpp

```diff
@@ -2153,6 +2153,8 @@ static bool MCGContextApplyPaintSettingsToSkPaint(MCGContextRef self, MCGColor p
 			case kMCGImageFilterHigh:
                 r_paint . setFilterLevel(SkPaint::kHigh_FilterLevel);
 				break;
+            default:
+                break;
```

Comment:
In this case, the enumeration definition is wrong and this `default` case shouldn't be needed. `kMCGImageFilterCount` shouldn't be a member of the `MCGImageFilter` enumeration for exactly this reason.